src/redis-helper.ts

```diff
@@ -145,6 +207,15 @@ export async function getTopTracksForArtist(redisClient: redis.RedisClient,
             });
         }
 
+        // set the last updated timestamp
+        redisClient.hmset(`artist:${artist.id}`,
+                          {top_track_ids_last_updated : JSON.stringify(new Date())},
```

Comment:
God I hate native node dates so much, I should really refactor this whole project to use moment.js or another one of the date libraries.

Does stringifying a date insert it as the epoch ms? I think that's how you're pulling it out

Reply:
oh no now I see, you pass the epoch ms from Date.now() to another Date() constructor to transform it to a date, and when you pull the full date string out you also pass it to a Date() constructor. Absolutely mental that this is the easiest way to do it. It also looks like calling JSON.stringify on a Date converts it to the the ISO format while just Date.toString doesn't.

I definitely want it in ISO, but will you specifically use `toISOString()` before stringifying it just for my sanity's/clarity's sake please? So a variable right before this `const now = new Date()` and then `top_track_ids_last_updated : JSON.stringify(now.toISOString())`. Or call the `toISOString` func in the variable declaration too and just use the `now` variable in the stringify call, whichever way you'd prefer

Reply:
Yup, I'm not familiar with Node and had to do a bit of reading and experimentation to get this working, but did eventually get there. I agree moment.js or an alternative would be much nicer though up to you if you want to do that wide a refactor! 

Will change to ISO date as requested